scripts/ci/podman-test.sh

```diff
@@ -35,50 +35,51 @@ make install
 
 # overlaysfs behaves differently on Ubuntu and breaks CRIU
 # https://bugs.launchpad.net/ubuntu/+source/linux/+bug/1857257
-podman --storage-driver vfs info
+PODMAN="podman --storage-driver vfs"
```

Comment:
Setting the storage-driver with an environment variable ([`export STORAGE_DRIVER=vfs`](http://docs.podman.io/en/latest/markdown/podman.1.html#storage-driver-value)), instead of adding the command-line option everywhere, might be a cleaner solution.

Reply:
Yes, much simpler. Thanks. Changed.